react/.eslintrc

```diff
@@ -0,0 +1,19 @@
+{
+  "extends": "vtex",
```

Comment:
Setup the right eslint
https://docs.google.com/document/d/1ePi5Mn3pe1dOorKrMtC-MclBDiIDimVzf0Uj61W8lA4/edit